src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequence.scss

```diff
@@ -0,0 +1,53 @@
+@import 'src/styles/common';
+
+$border: 1px solid $grey;
+
+.inputSequenceBox {
+  width: 860px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 28px 0 30px;
+  min-height: 50px;
+  border: $border;
+  border-bottom: none;
+}
+
+.body {
+  display: grid;
+  grid-template-columns: 1fr auto;
+  padding: 10px 65px 12px 28px;
+  background-color: $light-grey;
+  border: $border;
+
+  // FIXME: this is a hack
+  [class*='defaultUpload'] {
```

Comment:
Can this not be passed as a prop?

Reply:
We don't have to worry about this now as the styles for the upload component will be changing soon.

Reply:
This is a temporary hack anyway. The upload component that is targeted by this selector will be updated by Andrea; so I expect this css rule to be deleted then.